templates/landing/landing.css

```diff
@@ -0,0 +1,24 @@
+.landing {
+    h1 {
+        margin-top: 0;
+    }
+
+    h2 {
+        font-size: var(--heading-font-size-s);
+        margin-top: 1.25em;
+        line-height: 1.25;
+    }
+
+    h3 {
```

Comment:
All looks good, but the h3 that needs to be red 
https://www.learninga-z.com/site/resources/foundations-a-z-product-orientation

Reply:
YES, you are correct! I had this working at one point, didn't notice it stopped. I'll fix.